app/src/components/market/sections/market_pooling/market_pool_liquidity.tsx

```diff
@@ -179,12 +228,20 @@ const MarketPoolLiquidityWrapper: React.FC<Props> = (props: Props) => {
 
   const hasEnoughAllowance = RemoteData.mapToTernary(allowance, allowance => allowance.gte(amountToFund || Zero))
   const hasZeroAllowance = RemoteData.mapToTernary(allowance, allowance => allowance.isZero())
-
   const poolTokens = calcPoolTokens(
     amountToFund || Zero,
     balances.map(b => b.holdings),
     totalPoolShares,
   )
+  let displayPoolTokens = poolTokens
+  let displayTotalPoolShares = totalPoolShares
+  let baseCollateral = collateral
+  if (collateral.symbol.toLowerCase() in CompoundTokenType) {
+    const baseCollateralSymbol = getBaseTokenForCToken(collateral.symbol.toLowerCase()) as KnownToken
+    baseCollateral = getToken(networkId, baseCollateralSymbol)
+    displayPoolTokens = compoundService.calculateCTokenToBaseExchange(baseCollateral, poolTokens)
+    displayTotalPoolShares = compoundService.calculateCTokenToBaseExchange(baseCollateral, displayTotalPoolShares)
+  }
```

Comment:
atm if I create a cDai market with 1 Dai worth of liquidity it shows 0 pool tokens:

<img width="579" alt="Screen Shot 2021-01-27 at 11 09 41 am" src="https://user-images.githubusercontent.com/72715177/105923274-3c68f280-6090-11eb-8bad-a453c0e43a50.png">

Reply:
Couldn't reproduce this on my last few attempts, Should be fixed now.